docusaurus/docs/create-a-plugin/develop-a-plugin/best-practices.md

```diff
@@ -21,6 +21,8 @@ Is something missing from this list? [Let us know](https://github.com/grafana/pl
 - **Verify that your data source or app plugin can be provisioned.** Refer to [Provisioning](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) for more information.
 - **Include default dashboards in your data source or app plugin.** Refer to [Bundling of dashboards](https://grafana.com/developers/plugin-tools/introduction/plugin-types-usage#bundling-of-dashboards) for more information.
 - **Ensure that the minimum version of Grafana is correct.** Make sure that the `grafanaDependency` in your `plugin.json` points to the earliest version of Grafana that your plugin fully supports.
+- **Avoid using `console.log` in your plugin.** Console messages are usually for debugging purposes and therefore not suitable to ship to the client.
+- **Utilize log levels and never log sensitive information.** Make sure to utilize log levels properly, avoid excessive logging and never log credentials or other sensitive information.
```

Comment:
can we merge this info with what's in lines 60-62?

Reply:
I have merged these two tips in https://github.com/grafana/plugin-tools/pull/609